### DISCUSSION TITLE
Hi Rector Team I think I might have found a typo at the end of how_it_works.md

### DISCUSSION THREAD

**Discussion by alexandreelise** [rectorphp/rector#6827] [General]
Hi @TomasVotruba ,
I think I might have found a typo but the github actions bot rejected my []pull request](https://github.com/rectorphp/rector/pull/6826) and said that the code is generated from [rectorphp/rector-src](https://github.com/rectorphp/rector-src). But when I go there and I search for the same file how_it_works.md I couldn't find it. Am I missing something? Any feedback would be helpful. Just trying to be helpful.

**Comment by samsonasik** [comment]
You may looking for this file https://github.com/rectorphp/rector-src/blob/main/build/target-repository/docs/how_it_works.md

**Reply by alexandreelise** [reply]
Thanks @samsonasik That's what I was looking for. For some reason could'nt find it directly using github built-in find files feature.
Thanks again. Will do the correct pull request on that file and since mostly everything is automated it should be ok.

**Comment by alexandreelise** [comment]
Thanks @samsonasik Did my first pull request for rector-src. Actually it's just the docs but should be useful. Here it is [https://github.com/rectorphp/rector-src/pull/1293](https://github.com/rectorphp/rector-src/pull/1293)
